commands/stats.js

```diff
@@ -9,14 +9,19 @@ module.exports = {
     async execute(interaction) {
         const response = `
 __**global**__
-${await database.Player.count()} people have pinged at least once
-${await database.Player.sum('totalScore')} total pts have been gained
-${await database.Player.sum('score')} pts are currently owned
-${await database.Player.sum('clicks')} pings have been dealt with
+${await formatNumber(database.Player.count())} people have pinged at least once
```

Comment:
```suggestion
${formatNumber(await database.Player.count())} people have pinged at least once
```
await needs to be in front of all of the database commands, not the entire function thingy
(also change all ones below)